src/requests-and-replies.md

```diff
@@ -117,6 +123,9 @@ If a complete reply message is not received within this time, the request messag
 retransmitted. When retransmitting, different packet routes should be used, and new SURBs should be
 generated.
 
+// TODO maybe add: it is of for a mixnet node to disconnect a non mixnet node that don t respect these
+numbers. (even if it seems logic)
```

Comment:
I intend to implement detection and avoidance of bad mixnodes. This needs to be done carefully to ensure it can't be used to defeat anonymity. Once I have implemented something I'll document it here.